### DISCUSSION TITLE
[Verticapy 0.12.0] vDataFrame.get_len[] method behavior

### DISCUSSION THREAD

**Discussion by rbensa** [vertica/VerticaPy#511] [Q&A] [answered]
Hello,

The get_len function method only displays the length of each elements but not other corresponding elements (Other columns), is it a normal behavior ?

Kind regards,

Réda

**Comment by oualib** [comment]
Hi Réda,
I do not understand your question. Can you be more specific?

**Comment by rbensa** [comment]
Hello,

This what I get when I use the get_len function :

![image](https://user-images.githubusercontent.com/122989209/231111934-0fe69913-4ef3-4706-b490-c32afdf831a9.png)

It displays only the length of each element but not the correspondence (name).

Kind regards,

Réda

**Reply by oualib** [reply, marked as answer]
Indeed, it is a 'get' kind of function. You can affect this value to your vDF if you want to see the corresponding names: 
```
vDF["name_len"] = vDF["name"].get_len()
```
